$OpenBSD: patch-move_c,v 1.1 2012/05/05 14:03:15 sthen Exp $

From Debian:
  "Fix a bug in the ) vi command that could cause a segmentation
  fault (patch from elvis-almost-2.2.1)"

--- move.c.orig	Tue Oct 21 04:32:25 2003
+++ move.c	Sat May  5 13:53:51 2012
@@ -1620,6 +1620,7 @@ RESULT m_fsentence(win, vinf)
 	long	para;
 	long	offset;
 	long	count;
+	MARKBUF	tmp;
 
 	DEFAULT(1);
 
@@ -1651,6 +1652,7 @@ RESULT m_fsentence(win, vinf)
 	else
 	{
 		para = 0;
+		scanseek(&cp, marktmp(tmp, markbuffer(win->state->cursor), 0L));
 	}
 	marksetoffset(win->state->cursor, offset);
 	if (para == offset)
